File: py_basic/template.py
import sys

# 1. 빠른 입력 설정 (매번 sys.stdin.readline 쓰기 귀찮으므로 치환)
input = sys.stdin.readline
# 맨 끝의 개행문자까지 같이 입력받으므로 문자열을 저장하고 싶은 경우 오른쪽 공백을 제거하는 .rstrip()을 추가로 해준다.

# 2. 정수 하나 입력받기
n = int(input())

# 3. 공백으로 구분된 정수 여러 개 입력받기
a, b, c = map(int, input().split())

# 4. 정수 리스트 입력받기
arr = list(map(int, input().split()))

# 5. 2차원 배열 입력받기 (n행)
matrix = [list(map(int, input().split())) for _ in range(n)]

# 6. 문자열 입력받기 (개행문자 제거 필수)
s = input().strip()

File: js_basic/template.js
// 구현 => 지나친 추상화 피하기(빠른 완성이 우)
// 큐 => 인덱스 포인터 우선 고려
// return 감각보다 stdin/stdout 감각 우선
// 샘플 통과 직후 바로 제출하지말고 직접 만든 반례 먼저 돌리기

// 반례 3개를 먼저 적고
// 샘플 통과 후 내가 만든 테스트 3개를 더 돌리고
// 틀렸으면 “놓친 반례가 뭐였는지”를 한 줄로 남기기

// 기본 입력 템플릿

// 한줄로 입력
const fs = require("fs");
const input = fs.readFileSync(0, "utf8").trim().split("\n");

// const input = require('fs').readFileSync('/dev/stdin').toString().trim().split('\n');

let line = 0;
const nextLine = () => input[line++];

const [N, M] = nextLine().split(" ").map(Number);
const arr = nextLine().split(" ").map(Number);

console.log(N, M);
console.log(arr);
